.expeditor/release.omnibus.yml

```diff
@@ -66,8 +66,8 @@ builder-to-testers-map:
     - ubuntu-16.04-x86_64
     - ubuntu-18.04-x86_64
     - ubuntu-20.04-x86_64
-  windows-2012r2-i386:
-    - windows-2012r2-i386
+  # windows-2012r2-i386:
+  #   - windows-2012r2-i386
   windows-2012r2-x86_64:
```

Comment:
can we remove the commented code? We will still know what was removed from this commit if anything is needed in future

Reply:
Great point. Thanks. Doing that now.